core/server/common/src/test/java/alluxio/master/journal/raft/RaftJournalTest.java

```diff
@@ -85,6 +85,12 @@ public void suspendCatchupResume() throws Exception {
 
     // Suspend follower journal system.
     mFollowerJournalSystem.suspend();
+    try {
+      mFollowerJournalSystem.suspend();
+    } catch (Exception e) {
+      // Expected to fail when suspending a suspended journal.
```

Comment:
If this is expected to fail, then we should add an `assert.Fail()` after the `suspend` command.